conans/client/tools/system_pm.py

```diff
@@ -169,11 +170,12 @@ def install_packages(self, packages, update=True, force=False, arch_names=None):
                                   % "\n".join(packages))
                 return
 
-            if mode == "verify" and not self._installed(packages):
+            if mode == "verify" and self._to_be_installed(packages):
```

Comment:
This might not be ok, as in this context the ``packages`` is a list of alternate package names, not about different packages. So ``to_be_installed()`` will always return something if a list of names is provided.

I'd say with the ``return`` fix, it should be ok, can you please check? Thanks!

Reply:
Really? I thought install_packages was made to install all the packages https://docs.conan.io/en/latest/reference/conanfile/methods.html#systempackagetool

Reply:
Oh, ok, my bad, I was confused by the Github diff, didn't realize this was inside the ``install_packages``. 
You are right, this is ok.